### PR TITLE
Created README.md file and linters configuration

### PR DESCRIPTION
**Pull request by cuauh-cabrera**
Below you can find the new features included in this branch:

1.- Add an HTML file.

2.- Add a CSS file.

3.- Add README.md file.

4.- Add linters.

### CODE REVIEW

File: README.md
Comment (Felix45) [inline README.md:1]: - [Required] I would like to suggest that you ensure that all the compulsory sections of the readme are included in your readme file. The following sections should be present and filled in correctly, only the `Faq` section is optional. Kindly include any missing sections in your readme file.

- [ About the Project](https://github.com/microverseinc/readme-template#about-project)
  - [ Built With](https://github.com/microverseinc/readme-template#built-with)
    - [Tech Stack](https://github.com/microverseinc/readme-template#tech-stack)
    - [Key Features](https://github.com/microverseinc/readme-template#key-features)
   - [ Live Demo](https://github.com/microverseinc/readme-template#live-demo)
- [ Getting Started](https://github.com/microverseinc/readme-template#getting-started)
  - [Setup](https://github.com/microverseinc/readme-template#setup)
  - [Prerequisites](https://github.com/microverseinc/readme-template#prerequisites)
  - [Install](https://github.com/microverseinc/readme-template#install)
  - [Usage](https://github.com/microverseinc/readme-template#usage)
  - [Run tests](https://github.com/microverseinc/readme-template#run-tests)
  - [Deployment](https://github.com/microverseinc/readme-template#triangular_flag_on_post-deployment)
- [ Authors](https://github.com/microverseinc/readme-template#authors)
- [ Future Features](https://github.com/microverseinc/readme-template#future-features)
- [ Contributing](https://github.com/microverseinc/readme-template#contributing)
- [ Show your support](https://github.com/microverseinc/readme-template#support)
- [ Acknowledgements](https://github.com/microverseinc/readme-template#acknowledgements)
- [ FAQ (OPTIONAL)](https://github.com/microverseinc/readme-template#faq)
- [ License](https://github.com/microverseinc/readme-template#license)

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @cuauh-cabrera , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark:  
- [x] Good descriptive commit messages :white_check_mark:  
- [x] Descriptive pull request title and summary :white_check_mark: 

## Required Changes ♻️
- [Required] Please consider fixing any linter check that is failing on this pull request, a good pull request should always have all the linter checks passing with green checkmarks. When linter checks are passing it's a mark that your code follows industry standard guidelines. 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cuauh-cabrera** [comment]
Hi @Felix45 !

Thanks for your valuable feedback on this. 

After checking the linters configuration and trying to solve the issue with my coding partners, I could not able to set up properly linters in this repo. I tried every solution available and nothing solved the issues. After reviewing some articles and threads on StackOverflow and other sources, I noticed that the problem was related to a dependency not recognizing my OS (I’m working on MacOS Monterrey) and that the solution to that issue was to change some lines in the package.json and package.json lock files. However, those procedures are not recommended because we can break something else. After deliberating with my coding partners and other fellows, I decided to repeat the project in a new repo to determine if the error was on my side (some steps missing) or if the problem was related to my OS. After repeating the process in a new repo I found that I omitted to run the "npm init -y" command at the beginning of the process, and that omission caused all the issues. 

According to what was mentioned above, I was wondering if I could re-submit the project as a new repo, as I haven’t been able to fix the issue on this one. 

I’ll appreciate your understanding and support on this.

Best,

Cuau.

**Comment by Felix45** [comment]
Hi @cuauh-cabrera 
I think it is ok but please make sure that you mention this Pull request in the new pull request that you'll create and also remember to implement all the changes I had requested.

**Pull request closed without merging**